components/alert/alert-toast.js

```diff
@@ -200,6 +252,26 @@ class AlertToast extends LitElement {
 		clearTimeout(this._setTimeoutId);
 	}
 
+	_handleAlertClose(e) {
+		if (!e || e.target === this || !this.open) return;
+
+		const containerBottom = parseFloat(getComputedStyle(this._innerContainer).getPropertyValue('bottom'));
+		const closingContainerBottom = e.detail.bottom;
+		if (closingContainerBottom > containerBottom) return; // closing alert is above this alert, no need to adjust bottom spacing
+
+		this._bottomHeight -= e.detail.height;
+		this._bottomMargin -= 0.6;
```

Comment:
The `0.6` that gets used as the space between each toast feels important and shows up a few times. Might be worth throwing it in a constant at the top of the file so we don't accidentally forget one of them if we adjust it in the future?